.env

```
@@ -0,0 +1,16 @@
+
+
+# This was inserted by `prisma init`:
+# Environment variables declared in this file are NOT automatically loaded by Prisma.
+# Please add `import "dotenv/config";` to your `prisma.config.ts` file, or use the Prisma CLI with Bun
+# to load environment variables from .env files: https://pris.ly/prisma-config-env-vars.
+
+# Prisma supports the native connection string format for PostgreSQL, MySQL, SQLite, SQL Server, MongoDB and CockroachDB.
+# See the documentation for all the connection string options: https://pris.ly/d/connection-strings
+
+SUPABASE_URL=https://tkgihjsgtzjjrbtlxihm.supabase.co
+SUPABASE_KEY=sb_publishable_VGNWgkFdkuHUKZIB2JrPkA_N-8KmcoQ
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SUPABASE_KEY key should go before the SUPABASE_URL key

(UnorderedKey)